config/database.yml.example

```diff
@@ -1,28 +0,0 @@
-default: &default
-  adapter: postgresql
-  encoding: unicode
-  # http://guides.rubyonrails.org/configuring.html#database-pooling
-  pool: <%= ENV.fetch('RAILS_MAX_THREADS') { 5 } %>
-
-development:
-  <<: *default
-  database: agendamento_vacina_development
-  username: <%= ENV.fetch('POSTGRESQL_USERNAME') { 'postgres' } %>
-  password: <%= ENV.fetch('POSTGRESQL_PASSWORD') { '' } %>
```

Comment:
how did the official file looks like?
I added this:

```
host: postgres_agendamento_vacina
```

so I could find the databse

Reply:
It used simply `localhost`, which wasn't working on docker. You've probably needed this because you set the container name to `postgres_agendamento_vacina`. I was simply using `postgres`, but it's a better idea to define it like you did. I'll check if it works for me and commit updates

Reply:
It is a little odd that we need to name the container. It should follow the `<folder>_<service>_<id>` convention by default. There should be no problem to connect to it as `localhost` aswell, since we are binding the port.

Reply:
it didn't work for me when it was only localhost, even before I defined a container_name :thinking:

Reply:
Sorry, my mistake. It is `localhost` if we call it outside of the docker network. Since we are connecting from within the network, it is `postgres` (the service name). I fixed it.

Reply:
I simplified the `docker-compose.yml` file a little bit. It should work now. I removed the network definition and the `container_name`. Container names should respect the convention now.

I also ignored `database.yml` again because to me it is an anti-pattern. Some people prefer to have their one local configurations, setting passwords and so on. I changed `database.yml.example` host to `postgres` to make it work inside the docker network.

Reply:
Nice! thanks!